py/tests/integration/setup.yaml

```diff
@@ -73,6 +73,22 @@ tests: # test specific settings
         alice: dev1
       blocks_to_mine: 7
       spend_tx:
-        bob_pubkey: bob_address
+        bob_pubkey: Ym9iX2FkZHJlc3M= 
```

Comment:
I see this is bob_address in base64. This may be fragile in the future if length of pubkey checked though not in scope for this PR.

```
1> base64:decode("Ym9iX2FkZHJlc3M=").
<<"bob_address">>
```